Add styling for screenshots

ctauchen opened the pull request:
<!--- PR title format: [GH#<gh-issue-id>][DOCS#<jira-issue-id>]: <short-description-of-the-pr> --->
This component adds border styling for screenshot images. Previously,
screenshot images had ill-defined edges where whitespace bled into the
page background. This border helps make screenshots appear as coherent
elements of the page.

### New screenshot
![image](https://github.com/user-attachments/assets/7347e4f1-53fa-4563-9a5b-b2852102a115)
### Old screenshot
![image](https://github.com/user-attachments/assets/6fbf283b-0024-446f-8f30-ba7500e7a1c8)

Product Version(s):
<!--- Specify which versions of Calico, Calico Enterprise, and Calico Cloud your PR applies to. -->

Issue:
<!--- Add a link to the Jira ticket or GitHub issue, if applicable. --->

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

SME review:
- [ ] An SME has approved this change. 
<!--- SME approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

DOCS review:
- [ ] A member of the docs team has approved this change.
<!-- The Docs team must review and approve all changes. -->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

Merge checklist:
<!--- Mandatory for docs team members before merging code --->
- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed 


<!--- After you open your PR, ask for review from docs team:
  For community authors: tag @tigera/docs to ask for a review when your PR is ready --->

ctauchen committed:
Demonstrate screenshot styling

ronanc-tigera commented on src/___new___/components/Screenshot/index.tsx:
```diff
@@ -0,0 +1,13 @@
+export default function Screenshot({ src, alt }: ScreenshotProps) {
+  return (
+
+      <img src={src} alt={alt} className="screenshot" />
```
if you can, use Chakra components where possible https://v2.chakra-ui.com/docs/components/image/usage
Then, you can change your styles to a ts file and also use colors from the chakra theme

ronanc-tigera commented on src/___new___/components/Screenshot/index.tsx:
```diff
@@ -0,0 +1,13 @@
+
+export default function Screenshot({ src, alt }: ScreenshotProps) {
+  return (
+
```
Make sure to format the code

ronanc-tigera commented on src/___new___/components/Screenshot/index.tsx:
```diff
@@ -0,0 +1,13 @@
+  alt: string;
+};
+
+export default function Screenshot({ src, alt }: ScreenshotProps) {
```
the convention we follow is 
```
type ScreenshotProps = {
  src: string;
  alt: string;
};

const Component: React.FC<ScreenshotProps> = ({ src, alt}) => {
// logic here
}

export default Component
```

ronanc-tigera commented on src/___new___/components/Screenshot/styles.css:
```diff
@@ -0,0 +1,7 @@
+    border-radius: 8px;
+    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
+    background-color: white;
+    padding: 5px;
```
I think 8 or 12px of padding would look better

ronanc-tigera replied:
actually, looks better with no padding at all 
![image](https://github.com/user-attachments/assets/6cdefee4-fa8c-41f4-bfd4-3b36fa7b2da5)

ctauchen committed:
Add styling for screenshot

This component adds border styling for screenshot images. Previously,
screenshot images had ill-defined edges where whitespace bled into the
page background. This border helps make screenshots appear as coherent
elements of the page.

ctauchen commented:
@ronanc-tigera Changes made. Anything else?
